dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblem.java

```diff
@@ -28,15 +30,18 @@
   private AccessModifier modifier;
 
   InaccessibleClassProblem(
-      ClassFile sourceClass, ClassFile targetClass, Symbol classSymbol, AccessModifier modifier) {
+      ClassFile sourceClass,
+      @Nullable ClassFile targetClass,
+      ClassSymbol classSymbol,
```

Comment:
It was a generic `Symbol`. But InaccessibleClassProblem's symbol should always be a class symbol as written in the parameter name.